### DISCUSSION TITLE
How exactly would you use <ScreenStackHeaderConfig> to customize header?

### DISCUSSION THREAD

**Discussion by dhanushm360** [software-mansion/react-native-screens#834] [Q&A] [answered]
I'm having trouble listing properties within that component. I've tried to put it inside a screen component, but it wasn't working

**Comment by WoLewicki** [comment, marked as answer]
There are two scenarios. First, if you use `native-stack`, this component should not be used directly, but is controlled via props of the screens (mainly those which start with `header...`). Second, if you want to build your own navigators, then the `ScreenStackHeaderConfig` should be a direct child of the `Screen` component in `ScreenStack` (see https://github.com/software-mansion/react-native-screens/blob/master/src/native-stack/views/NativeStackView.tsx#L132 which resolves to `ScreenStackHeaderConfig`). Does it solve your problem?
